framework_agreement/model/purchase.py

```diff
@@ -1,8 +1,5 @@
-# -*- coding: utf-8 -*-
```

Comment:
My reasoning was that this is not dangerous because in python 2 the default is ASCII so that if Amélie or Joël contribute to that file, their name will not be garbled (which is bad), but they will get a nice error explaining what to do.

I like to get rid of stuff if the default is good enough, but will put that back if you insist :cat: